main/appengine_config.py

```diff
@@ -6,6 +6,10 @@
 if os.environ.get('SERVER_SOFTWARE', '').startswith('Google App Engine'):
   sys.path.insert(0, 'lib.zip')
 else:
+  if os.name == 'nt':
+    os.name = None
+    sys.platform = ''
```

Comment:
No problem!

Yeah, we probably should file an issue with Click as it is a part of Flask now, and I'm sure we're not the only ones who are going to run into this issue with App Engine...in which case, it would probably be good to file one with Flask too.